tslint.json

```diff
@@ -20,15 +25,18 @@
             "spaces"
         ],
         "interface-over-type-literal": true,
+        "interface-name": false,
+        "member-ordering": false,
         "new-parens": true,
         "no-consecutive-blank-lines": [
             true,
             1
         ],
+        "no-console": false,
         "no-default-export": true,
-		"no-duplicate-variable": true,
-		"no-duplicate-imports": true,
-		"no-unused-variable": true,
+		    "no-duplicate-variable": true,
+		    "no-duplicate-imports": true,
+		    "no-unused-variable": true,
```

Comment:
There is no reason these 3 lines should have different indentation to the line above them. Can you fix that?

Reply:
looks like this is github displaying it false. Look how things look in vscode:
![image](https://user-images.githubusercontent.com/25126/56027233-f5243d00-5d15-11e9-975e-2227435087a0.png)

Reply:
Shows just like on github in my vscode. And the interesting part is that the neighbouring lines are indented by 8 spaces. And these 3 have 2 tabs and 4 spaces. So I guess you have your vscode show tabs as 2 spaces.

Reply:
@jrichter1 indeed, those 3 had tabs. But then I have vscode set to 4 spaces:
```
 "[typescript]": {
      "editor.tabSize": 4,
```

Reply:
weird enough, tslint config even has the rule that would enforce spaces and fix it. But it didn't seem to fix the above:
```
        "indent": [
            true,
            "spaces"
        ],
```